impl/src/bitfield.rs

```diff
@@ -180,6 +184,56 @@ impl BitfieldStruct {
         ))
     }
 
+    /// Get a list of fields, both in the form of a `String` as well as the corresponding faillible
+    /// getter method
+    fn get_field_strings_and_getters(&self) -> (Vec<String>, Vec<syn::Ident>) {
+        self.item_struct
+            .fields
+            .iter()
+            .enumerate()
+            .map(|(i, field)| {
+                let field_name = field
+                    .ident
+                    .as_ref()
+                    .map(syn::Ident::to_string)
+                    .unwrap_or_else(|| format!("get_{}", i));
```

Comment:
In this case we want to use the field's actual name as string, so we can strip away the `get_` prefix for unnamed fields.